zerver/lib/markdown/help_settings_links.py

```diff
@@ -114,16 +114,32 @@
 1. On the left, click {setting_reference}.
 """
 
+users_tab_markdown = """
+1. Click on the **gear** (<i class="zulip-icon zulip-icon-gear"></i>) icon in the upper
+   right corner of the web or desktop app.
+
+1. Select **{setting_type_name}**.
+
+1. On the left, click **Users**.
+
+1. Select the {setting_reference} tab.
+"""
+
 
 def getMarkdown(setting_type_name: str, setting_name: str, setting_link: str) -> str:
     if relative_settings_links:
         relative_link = f"[{setting_name}]({setting_link})"
         # The "Bots" label appears in both Personal and Organization settings
         # in the user interface so we need special text for this setting.
-        if setting_name in ["Bots", "Users"]:
+        if setting_name in ["Bots", "Users", "Invitations", "Deactivated"]:
             return f"1. Navigate to the {relative_link} \
                     tab of the **{setting_type_name}** menu."
```

Comment:
I think we still want all of these to say "the **Users** tab of the **Organization settings** menu." since there isn't a tab in the left side of that overlay for invitations or deactivated (users).